src/main/java/org/scijava/jupyter/notebook/DefaultNotebookService.java

```diff
@@ -56,9 +56,30 @@
     @Parameter
     private LogService log;
 
-    public DefaultNotebookService() {
-        Displayers.register(String.class, (Displayer) StringDisplayer.get());
-        Displayers.register(List.class, (Displayer) ListDisplayer.get());
+    @Parameter
+    private ConvertService converService;
+
+    /**
+     * Use the most appropriate output type according to the input object.
+     *
+     * @param object
+     * @return
+     */
+    @Override
+    public Object display(Object object) {
+
+        if (object.getClass() == String.class) {
```

Comment:
Good point. I'll use that. And for the user who want to select a specific output we can just create other methods such as `displayAsJPEG()`, `displayAsJSONTree()`, etc